From 1e6669d08c64546cb0fac011eabd551fd3a0a0af Mon Sep 17 00:00:00 2001
From: Adam Sampson <ats@offog.org>
Date: Tue, 3 Apr 2018 23:48:43 +0100
Subject: [PATCH i-g-t] Use flex -i when building with meson.

When built with automake, the assembler's tokeniser is built with -i,
making it case-insensitive. This wasn't being done with meson.

(The symptom was build failures in intel-vaapi-driver, which uses
register types like ":ub". In lex.l, they're written as ":UB", so the
lexer was rejecting them without -i.)

Signed-off-by: Adam Sampson <ats@offog.org>
Reviewed-by: Arkadiusz Hiler <arkadiusz.hiler@intel.com>
---
 assembler/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/assembler/meson.build b/assembler/meson.build
index 7517d82f..c858bc0f 100644
--- a/assembler/meson.build
+++ b/assembler/meson.build
@@ -19,7 +19,7 @@ bison = find_program('bison')
 
 lgen = generator(flex,
 		output : '@BASENAME@.c',
-		arguments : ['-o', '@OUTPUT@', '@INPUT@'])
+		arguments : ['-i', '-o', '@OUTPUT@', '@INPUT@'])
 
 lfiles = lgen.process('lex.l')
 
-- 
2.17.0

